tests/general/QuotesBearTest.py

```diff
@@ -0,0 +1,99 @@
+import unittest
+from queue import Queue
+from textwrap import dedent
+
+from coalib.results.HiddenResult import HiddenResult, Result
+from bears.general.QuotesBear import QuotesBear
+from coalib.results.SourceRange import SourceRange
+from coalib.settings.Section import Section
+from coalib.testing.LocalBearTestHelper import verify_local_bear, execute_bear
+
+
+class QuotesBearDiffTest(unittest.TestCase):
+
+    def setUp(self):
+        self.section = Section('')
+        self.uut = QuotesBear(self.section, Queue())
+
+        self.double_quote_file = dedent("""
+        '''
+        Multiline string
+        '''
+        "a string with double quotes!"
+        'A single quoted string with " in it'
+        """).splitlines(True)
+
+        self.single_quote_file = dedent("""
+        '''
+        Multiline string
+        '''
+        'a string with single quotes!'
+        "A double quoted string with ' in it"
+        """).splitlines(True)
+
+        self.filename = 'f'
+
+        self.dep_results = {
+            'AnnotationBear':
+                [HiddenResult(
+                    'AnnotationBear',
+                    {'comments': (), 'strings': (
+                        SourceRange.from_values(self.filename, 2, 1, 4, 3),
+                        SourceRange.from_values(self.filename, 5, 1, 5, 30),
+                        SourceRange.from_values(self.filename, 6, 1, 6, 37))
+                     }
+                )]
+        }
+
+    def test_error_handling(self):
+        dep_results = {'AnnotationBear': [Result("test", "test")]}
+        with execute_bear(self.uut, self.filename, self.double_quote_file,
+                          dependency_results=dep_results) as results:
+            self.assertEqual(len(results), 0)
+
+        dep_results = {'AnnotationBear': [HiddenResult('a', 'error!')]}
+        with execute_bear(self.uut, self.filename, self.double_quote_file,
```

Comment:
this is the cse where there is no string in the file right ? maybe it should go in a different test function (this is just a preference :) )